src/main/java/core/basesyntax/Application.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+import model.Figure;
+import utility.FigureSupplier;
+
+public class Application {
+    private static final int MAX_LENGTH_ARRAY = 6;
+    private static final FigureSupplier figureSupplier = new FigureSupplier();
```

Comment:
According to the checklist, you should not create an instance of `FigureSupplier` as a static field. Instead, create it inside the `main` method before the loop starts.